frontend/lib/evictionfree/tests/site.test.tsx

```diff
@@ -20,4 +20,14 @@ describe("EvictionFreeSite", () => {
       )
     );
   });
+
+  it("renders special text on home page when EvictionFree is suspended", async () => {
```

Comment:
This is great! Do you want a test that checks that if EFNY is not suspended, it renders some other specific test?

Reply:
Yes! So errr, the `renders home page` test right above this does exactly that, but the name is too vague. Let me change that.